addon/components/horizontal-list-swiper/no-delay-on-transitions-in-test/component.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import layout from './template';
+import {getOwner} from '@ember/application';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  layout,
+  window: window,
```

Comment:
Should use the window service if open sourced

Reply:
yes